### ISSUE TITLE
New Room List: Very hard to read due to transparency on background image

### ISSUE THREAD

**Issue by MacLemon** [element-hq/element-web#14493]
### Description
The new room list places all text on multiple semi-transparent layered cells upon a blurred background image which makes most text very hard, if not impossible to read.

### Steps to reproduce

- Upgrade to 1.7.0
- See new room list
- There is no step three

Describe how what happens differs from what you expected.

Logs being sent: no, UI/UX/Accessibility problem

![Element 1 7 0 New Room list transparency](https://user-images.githubusercontent.com/90635/87575336-0eadf480-c6d0-11ea-9afe-9a0cdb8a88f9.png)

### Version information

- **Platform**: Desktop 1.7.0

For the desktop app:
- **OS**: macOS 10.13.6 (High Sierra)
- **Version**: 1.7.0

### Notes
- This is an accessibility regression from Riot where the room list was not semi-transparent and not on a blurry background image.
- After experimenting with my avatar icons it turns out, the background image there is actually a blurred version of my avatar icon. To make text legible I must change my avatar to a black square.
- macOS has a system wide setting to reduce transparency which will prevent exactly these kinds of text over semi-transparent mush to make text more legible. The Electron App does not respect that setting, nor does it provide a setting to turn off transparency by itself.
- This also affects the Community filter sidebar (if enabled).
- All that transparency makes using the room list very slow, it lags behind by at least half a second in good moments and several seconds in worse ones.

**Comment by stephanedupont** [comment]
It also leads to sidebars being darker and as a result kinda ugly (to my taste at least!) at all, see #14548

**Comment by turt2live** [comment]
Closing in favour of https://github.com/vector-im/riot-web/issues/14665

**Comment by turt2live** [comment]
oh, hmm, wrong issue sorry.

**Comment by SimonBrandner** [comment]
@turt2live, I am a little confused - [#14665](https://github.com/vector-im/element-web/issues/14665) seems to cover this issue. Am I missing something?

**Comment by turt2live** [comment]
14665 is about turning the feature off, not about improving accessibility when it is turned on.

**Comment by SimonBrandner** [comment]
Ah, that makes sense - sorry for annoying

**Comment by gsouquet** [comment]
Some changes have recently been made to the background blur.
@MacLemon could you check and see whether this has mitigated your a11y concerns and whether that improved performance with your current setup at all?

Looking whether we can close this issue or not

**Comment by MacLemon** [comment]
> 14665 is about turning the feature off, not about improving accessibility when it is turned on.

Turning this blurry mess off IS the best way to improve accessibility for everyone! Under any circumstance, blur is an accessibility nightmare for anyone without perfect eyesight.

> Some changes have recently been made to the background blur.
> @MacLemon could you check and see whether this has mitigated your a11y concerns and whether that improved performance with your current setup at all?
> 
> Looking whether we can close this issue or not
Sorry, this is not fixed. The *best* solution would be to get rid of that blurry avatar. It serves absolutely no purpose to the user and only has downsides with regards to accessibility. It still wastes render cycles and makes redraw noticeably sluggish. (The grade of blurryness doesn't make any difference for rendering, it still has to be calculated.)

Please, get rid of this thing, or at least give people a way to completely turn it off, and make the default OFF. Those who are eager to have their own avatar as blurry background can then turn it on if they absolutely desire.

**Comment by robintown** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/11079

**Comment by germain-gg** [comment]
We're keeping that behaviour for now, trying to minimise the impact of the changes.
The plan is to remove it when we get to rebuild the room list.

**Comment by Cyborgscode** [comment]
I know you will mark this as unwanted comment, but i testes element-desktop today, and that issue is more than anoying. I instantly reverted to schildichat, because it does not have this problem. 

I strongly advise to remove the background blurred avatar logo and make the sidebar in the same black as the chat area. That light greyish just looks bad. These CSS based changes are done in one minute. There is no reason to dealy this change any longer.

**Comment by t3chguy** [comment]
It's already removed in the new room list in labs on develop.element.io

**Comment by t3chguy** [comment]
Fixed by new room list
